backend/matching-service/app/src/matching/matching.gateway.ts

```diff
@@ -39,6 +45,11 @@ export class MatchingGateway {
       preferences: roomParams.preferences,
     };
     this.waiting.push(newWaiting);
+
+    setTimeout(() => {
+      this.leaveRoom(roomParams.user);
+    }, 30000);
```

Comment:
nit: this 30k can be a constant - is this also used elsewhere in this service? or just on the fe

Reply:
a5ec34d5c2f23122dbd1b049e506d11f08a8297f